pandas/tests/groupby/test_nth.py

```diff
@@ -691,6 +691,22 @@ def test_first_multi_key_groupbby_categorical():
     tm.assert_frame_equal(result, expected)
 
 
+@pytest.fixture()
+def small_df():
+    data = [
+        [0, "a", "a0_at_0"],
+        [1, "b", "b0_at_1"],
+        [2, "a", "a1_at_2"],
+        [3, "b", "b1_at_3"],
+        [4, "c", "c0_at_4"],
+        [5, "a", "a2_at_5"],
+        [6, "a", "a3_at_6"],
+        [7, "a", "a4_at_7"],
+    ]
+    df = DataFrame(data, columns=["Index", "Category", "Value"])
```

Comment:
can you avoid the use of the work Category as this implies a dtype, which i dont' think yo mean

Reply:
Done